starknet_py/tests/unit/hash/blake2s_test.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=line-too-long
+# fmt: off
+"""
+Test the encode_felt252_data_and_calc_blake_hash function
+with the same results as the Cairo v0.14 implementation.
+Reference: https://github.com/starkware-libs/cairo-lang/releases/tag/v0.14.0
```

Comment:
For me it makes more sense to simply add a permalink to blockifier and say that the tested values are taken from there.